web/src/narrow.js

```diff
@@ -86,7 +86,7 @@ export function changehash(newhash, trigger) {
     browser_history.set_hash(newhash);
 }
 
-export function save_narrow(terms, trigger) {
+export function set_hash_to_filter_terms(terms, trigger) {
```

Comment:
Maybe `update_hash_to_match_filter` would be a better name? And then maybe we should just pass `filter, trigger` as the parameters?

Reply:
Yeah, updated!